securedrop/journalist_templates/_sources_confirmation_modal.html

```diff
@@ -0,0 +1,32 @@
+<div id="{{ modal_data.modal_id }}" class="menu-modal-dialog">
+  <a href="#close" class="external"></a>
+  <div id="delete-menu-dialog">
+    <div id="delete-menu-no-select">
+      <p class="modal-danger-text">
+      <span class="modal-danger-text">{{ gettext("Nothing Selected") }}</span>
+      </p>
+      <p>
+      {{ gettext("You must select one or more items for deletion.") }}
+      </p>
+    </div>
+    <div id="delete-menu-cta">
+    <p>
+      <span id="delete-menu-summary"></span>
+    </p>
+    <p>
+    {{ gettext("What would you like to delete?") }}
+    </p>
+    <p>
+      <button id="delete-files-and-messages" type="submit" name="action" value="delete-data" class="small btn danger">{{ gettext('Messages and Files') }}</button>
+    </p>
+    <p>
+      <a href="#delete-sources-confirm-modal" id="delete-collections">
+        <button type="button" class="small danger btn">{{ gettext('Source Accounts') }}</button>
```

Comment:
This button appears to be at a lower width than the first button:
![Screenshot from 2021-02-17 16-26-25](https://user-images.githubusercontent.com/213636/108285922-04a81480-713d-11eb-8456-336e74813b4a.png)
![Screenshot from 2021-02-17 16-27-56](https://user-images.githubusercontent.com/213636/108286096-5355ae80-713d-11eb-8f1f-602cb871dbcd.png)

In the prototype, they have the same width, which IMO looks significantly cleaner in a stacked button layout:

Reply:
:heavy_check_mark: